### ISSUE TITLE
[Poster Creation Group] IPNS

### ISSUE THREAD

**Issue by mgoelzer** [libp2p/developer-meetings#15]
## IPNS

Your Name: 

Session Type (choose one):  
1. lightning (3-10 min)
1. 30-minute presentation on a specific topic
1. leader of a 2-3 person poster group (you should have knowledge of the poster topic)
1. leader/moderate of a design session

Title (a few words): 

Abstract (1 sentence up to a couple paragraphs, as you prefer):

**Comment by lgierth** [comment]
Is this meant for the IPFS meetings?

**Comment by mgoelzer** [comment]
@lgierth Probably, yes.  Can you think of a good libp2p topic to replace this?  Else, we can close this issue.

**Comment by mgoelzer** [comment]
There is discussion [here](https://github.com/ipfs/js-ipfs/pull/1400#issuecomment-399266349) that makes arguments both for and against IPNS being considered part of libp2p.

In light of this, I propose that we **do** include IPNS as a libp2p poster or presentation topic.  I've assigned it to @vasco-santos (who requested it in https://github.com/libp2p/developer-meetings/issues/10#issuecomment-400383196.  

@vasco-santos Can you fill in the issue above?  Pick between presentation and poster, and give it a title and short abstract for Sched.

**Comment by mgoelzer** [comment]
@vasco-santos But if you prefer, we can move this to the [IPFS meeting's sessions](https://github.com/ipfs/developer-meetings/tree/master/sessions) instead.  My point is really just that it's a good presentation, and if it straddles the line, it probably doesn't matter which event it is part of.  If it's important to you, though, let's move it.

**Comment by vasco-santos** [comment]
Thanks @mgoelzer ! I will fill this issue between today and tomorrow!

**Comment by vasco-santos** [comment]
@mgoelzer the description is updated. 

I decided to keep the **Poster Creation Group**  category, but I am also available for preparing a presentation if you have that preference.

**Comment by mgoelzer** [comment]
@vasco-santos Awesome - thank you!

The Poster Creation Group vs Presentation decision is up to you.  They are both valuable.  If you'd rather do a presentation, that would be fine.  Just update the issue.  But the PCG's will be fun and interactive for attendees, so that is the benefit of doing those.  Again, your call.

**Comment by vasco-santos** [comment]
yeah! I also agree that the poster creation group will be more interesting and interactive!

**Comment by hugomrdias** [comment]
Should we request here to be part of this poster group ?

**Comment by mgoelzer** [comment]
@hugomrdias Good question.  I had been thinking that the groups would form ad hoc, so I was not planning to create a list of members in advance.  Just one leader/moderator to anchor the group to ensure that there is a nucleus around which the group can form.  So, I do not think it's necessary.
